client/mmadness/src/central/vote/vote.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import './../central.css';
+
+export default class Vote extends Component {
+  render() {
+
+    const messageNoAuthor = this.props.messages
+      .filter(message => { 
```

Comment:
Keep it DRY.
I see you repeated this operation at least twice, do it once in the parent before rendering `Vote` or `Voting` and pass it down, so you won't repeat it.